pkg/pgmodel/end_to_end_tests/drop_test.go

```diff
@@ -381,3 +381,134 @@ func TestSQLDropMetricChunk(t *testing.T) {
 
 	})
 }
+
+// Tests case that all metric data was dropped and then the metric came back alive
+func TestSQLDropAllMetricData(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+	if !*useTimescaleDB {
+		t.Skip("This test only runs on installs with TimescaleDB")
+	}
```

Comment:
It looks like we run the DELETE independent of whether timescaledb is installed, should we have a vanilla prom test?